.github/workflows/build_slim_packages.yaml

```diff
@@ -30,12 +30,11 @@ jobs:
       matrix:
         profile:
         - ["emqx", "24.3.4.2-2", "el7"]
-        - ["emqx", "24.3.4.2-2", "ubuntu20.04"]
         - ["emqx", "25.1.2-2", "ubuntu22.04"]
-        - ["emqx-enterprise", "24.3.4.2-2", "ubuntu20.04"]
-        - ["emqx-enterprise", "25.1.2-2", "ubuntu22.04"]
+        - ["emqx-enterprise", "24.3.4.2-2", "amzn2"]
+        - ["emqx-enterprise", "25.1.2-2", "ubuntu20.04"]
```

Comment:
Q: should this be 22.04?

Reply:
Good to build test packages for both 20.04 and 22.04. Enterprise is more likely to use older systems.